doc/source/whatsnew/v0.23.0.txt

```diff
@@ -119,6 +119,59 @@ Current Behavior
 
     s.rank(na_option='top')
 
+
+.. _whatsnew_0230.enhancements.assign_dependent:
+
+``.assign()`` accepts dependent arguments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :func:`DataFrame.assign()` now accepts dependent kwargs. In earlier versions this throws a Keyerror exception anymore. (:issue: `14207)
```

Comment:
put a reference to the doc-section on ``.assign`` in the docs which also needs updating.

Reply:
Apologies, but what is the correct path to refer to? What should be the updated content of the `.assign` docs?